### PR TITLE
3 us 2 merchant dashboard links

### PR DESCRIPTION
**Pull request by KatieLonsdale**
## Describe Changes
- add tests for merchant dashboard links
- add link to items index page from merchant show page
- add link to invoices index page from merchant show page
- add invoices and items controllers
- add invoices and items index views
- added routes

## Type of change

- [x] Complete a User Story (tests all passing)
- [ ] Refactor 
- [ ] Bug fix 
- [ ] Other: (describe here!)

## Extra! Extra! Read all about it!
- We worked on this as a joyful pair

## Number of Reviews
- [x] one
- [ ] two
- [ ] EVERYONE PLS!

### CODE REVIEW

File: app/views/merchants/show.html.erb [hunk 1]
```diff
@@ -1 +1,9 @@
-<h1><%= @merchant.name %></h1>
\ No newline at end of file
+<h1><%= @merchant.name %></h1>
+
+<div id="My-Items">
```
Comment (grace-joh) [inline app/views/merchants/show.html.erb:3]: I think convention for html id attribute is all lowercase!

File: app/views/merchants/show.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
...
+  <%= link_to 'My Items', "/merchants/#{@merchant.id}/items" %>
+</div>
+
+<div id="My-Invoices">
```
Comment (grace-joh) [inline app/views/merchants/show.html.erb:7]: see first comment

File: spec/features/merchants/show_spec.rb [hunk 3]
```diff
@@ -13,5 +13,27 @@
       expect(page).to have_content(@merch_1.name)
       expect(page).to have_no_content(@merch_2.name)
     end
+
+    it 'displays a link to merchant items index' do
+      visit "/merchants/#{@merch_1.id}/dashboard"
+      within("#My-Items") do
```
Comment (grace-joh) [inline spec/features/merchants/show_spec.rb:19]: see first comment

File: spec/features/merchants/show_spec.rb [hunk 4]
```diff
@@ -13,5 +13,27 @@
...
+       expect(page).to have_link('My Items')
+
+       click_link('My Items')
+      end
+       expect(current_path).to eq("/merchants/#{@merch_1.id}/items")
+    end
+
+    it 'displays a link to merchant invoices index' do
+      visit "/merchants/#{@merch_1.id}/dashboard"
+      within("#My-Invoices") do
```
Comment (grace-joh) [inline spec/features/merchants/show_spec.rb:29]: see first comment!

**Review by grace-joh — CHANGES_REQUESTED** [review]
just a quick casing change pls!

File: spec/features/merchants/show_spec.rb (hunk 3 above)
Comment (KatieLonsdale) [reply spec/features/merchants/show_spec.rb]: changed!

**Review by grace-joh — APPROVED** [review]
looks great!

**Pull request merged**